requirements.txt

```diff
@@ -102,4 +102,5 @@ typing-extensions==3.10.0.0
 urllib3==1.26.6; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4' and python_version < '4'
 watchdog==2.1.3; python_version >= '3.6'
 wrapt==1.12.1
-zipp==3.5.0; python_version >= '3.6'
\ No newline at end of file
+zipp==3.5.0; python_version >= '3.6'
```

Comment:
not familiar with this idiom. Python v is specified separately?

Reply:
This was autogenerated after running `pipenv lock --requirements` that generates the `requirements.txt` content. I did this so I could use Anaconda to develop a virtual env.

Reply:
I see

Reply:
will you add to Makefile

Reply:
Done!